luxtronik/discover.py

```diff
@@ -32,35 +34,37 @@ def discover() -> list[(str, int)]:
 
         # send AIT magic broadcast packet
         server.sendto(LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode(), ("<broadcast>", port))
-        LOGGER.debug("Sending broadcast request %s", LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode())
+        LOGGER.debug(
+            "Sending broadcast request %s", LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode()
+        )
 
         while True:
             try:
-                res, con = server.recvfrom(1024)
-                res = res.decode("ascii", errors="ignore")
+                recv_bytes, con = server.recvfrom(1024)
+                res = recv_bytes.decode("ascii", errors="ignore")
                 # if we receive what we just sent, continue
                 if res == LUXTRONIK_DISCOVERY_MAGIC_PACKET:
                     continue
                 ip_address = con[0]
                 # if the response starts with the magic nonsense
                 if res.startswith(LUXTRONIK_DISCOVERY_RESPONSE_PREFIX):
-                    res = res.split(";")
-                    LOGGER.debug("Received response from %s %s", ip_address, str(res))
+                    res_list = res.split(";")
+                    LOGGER.debug(
+                        "Received response from %s %s", ip_address, str(res_list)
+                    )
                     try:
-                        port = int(res[2])
-                        if port < 1 or port > 65535:
-                            LOGGER.debug(
-                                    "Response contained an invalid port, ignoring"
-                                    )
-                            port = None
+                        res_port: int | None = int(res_list[2])
```

Comment:
Was this a change from flake8 also? Not sure if this really improves anything, since `port` is also pretty descriptive and shorter than `res_port`.

Reply:
The variable port was double used.
The port to scan and the API port reported by the heatpump answer.
I have created an own variable for the reported port.